src/components/pages/ImageClassification/ImageAnnotationModal/ImageAnnotationPopup/SelectAttributeFromClassifierGuesses.js

```diff
@@ -15,14 +29,33 @@ const isAClassifierGuessOfSelectedPoint = (annotations, ba_gr) =>
 
 const isOptionAlreadyAdded = (acc, value) => acc.some((option) => option.value === value)
 
-const ExistingRows = ({
+const SelectAttributeFromClassifierGuesses = ({
   selectedPoint,
   dataToReview,
   setDataToReview,
   setIsDataUpdatedSinceLastSave,
+  databaseSwitchboardInstance,
 }) => {
   const [selectedExistingRow, setSelectedExistingRow] = useState('')
 
+  const {
```

Comment:
I moved the logic to deal with the new attribute modal from the deleted `NewRow` component to a hook